README.md

```diff
@@ -46,29 +46,53 @@ The following is a list of all of the static color functions defined on `struct
 ```swift
 public static func turquoiseColor() -> OSColor!
 public static func greenSeaColor() -> OSColor!
+public static func mediumTurquoiseColor() -> OSColor! 
+public static func lightSeaGreenColor() -> OSColor!
 public static func emeraldColor() -> OSColor!
 public static func nephritisColor() -> OSColor!
+public static func gossipColor() -> OSColor!
+public static func salemColor() -> OSColor!
 public static func peterRiverColor() -> OSColor!
 public static func belizeHoleColor() -> OSColor!
+public static func riptideColor() -> OSColor!
+public static func dodgerBlueColor() -> OSColor!
 public static func amethystColor() -> OSColor!
 public static func wisteriaColor() -> OSColor!
+public static func lightWisteriaColor() -> OSColor!
+public static func plumColor() -> OSColor!
 public static func wetAsphaltColor() -> OSColor!
 public static func midnightBlueColor() -> OSColor!
+public static func hokiColor() -> OSColor!
+public static func ebonyClayColor() -> OSColor!
 public static func sunflowerColor() -> OSColor!
 public static func tangerineColor() -> OSColor!
+public static func confettiColor() -> OSColor!
+public static func capeHoneyColor() -> OSColor!
 public static func carrotColor() -> OSColor!
 public static func pumpkinColor() -> OSColor!
+public static func ecstasyColor() -> OSColor!
+public static func jaffaColor() -> OSColor!
 public static func alizarinColor() -> OSColor!
 public static func pomegranateColor() -> OSColor!
+public static func monzaColor() -> OSColor!
+public static func thunderBirdColor() -> OSColor!
 public static func cloudsColor() -> OSColor!
 public static func silverColor() -> OSColor!
+public static func galleryColor() -> OSColor!
+public static func ironColor() -> OSColor!
 public static func concreteColor() -> OSColor!
 public static func asbestosColor() -> OSColor!
+public static func pumiceColor() -> OSColor!
+public static func lynchColor() -> OSColor!
 ```
 
-![Colors Palette](https://github.com/brynbellomy/FlatUIColors/blob/master/ColorsPalette.png)
+![Colors Palette](https://github.com/avazquezpr/FlatUIColors/blob/master/ColorsPalette.png)
```

Comment:
Hey, would you mind changing this line back to point to this repo since it's the original?